auth/deployment.yaml

```diff
@@ -45,11 +45,6 @@ spec:
             value: "{{ default_ns.name }}"
```

Comment:
For some reason GitHub shows this file. I think if you merge/rebase with main it will go away.